components/TwListView.js

```diff
@@ -1,100 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { StyleSheet, ScrollView, Text, TextInput, View } from 'react-native';
+import { StyleSheet, SectionList, StatusBar, View } from 'react-native';
+import { Header, ListItem, Body, Item, Input, Right, Icon, Text } from 'native-base';
 
-/*
- * StyleSheet
- * textInput for search bar style
- * textBar between Title, Notice and Location
-*/
-const styles = StyleSheet.create({
-  container: {
-    borderWidth: 1.0,
-    backgroundColor: 'rgba(247,247,247,1.0)',
-    padding: 40,
-    height: 44,
-    flexDirection: 'column',
-
-  },
-  textInput: {
-    height: 30,
-    borderWidth: 1,
-    borderRadius: 8,
-    backgroundColor: 'transparent',
-    borderColor: '#cecece',
-    marginBottom: 5,
-    marginTop: 15,
-    fontWeight: 'bold',
-    marginHorizontal: 4,
-
-  },
-  top: {
-    height: 25,
-    backgroundColor: 'transparent',
-    borderRadius: 4,
-    justifyContent: 'space-between',
-    flex: 2,
-    flexDirection: 'row',
-  },
-  title: {
-    height: 25,
-    backgroundColor: 'steelblue',
-    fontWeight: 'bold',
-
-  },
-  text1: {
-    height: 28,
-    backgroundColor: 'skyblue',
-
-  },
-  text2: {
-    backgroundColor: 'powderblue',
+import { getNoticeType, groupBy, formatDate } from '../utils/helpers';
 
+const styles = StyleSheet.create({
+  header: {
+    paddingTop: StatusBar.currentHeight,
+    backgroundColor: 'white',
+    height: StatusBar.currentHeight + 60,
   },
-
 });
 
-// disabled for now, at least until we figure out if this
-// will have state or not
-/* eslint-disable react/prefer-stateless-function */
+/**
+ * @function
+ * Renders each List Item
+ */
+const TwListItem = (item) => {
+  const type = getNoticeType(item.NOTICETYPE[0]);
+  return (
+    <ListItem>
+      <Body>
+        <Text>{item.TITLE.split(' - ')[0]}</Text>
+        <Text style={type.getStyle()}>{type.getIcon()} {type.name} </Text>
+        <Text note numberOfLines={1}>{item.DESCRIPTION.replace(/<\/?[^>]+(>|$)/g, '')}</Text>
+      </Body>
+      <Right>
+        <Text note>{formatDate(new Date(item.STARTDATE))}</Text>
```

Comment:
Hi, 
why are you manually formatting the date here and on the item details view, you just say new Date(date).toLocaleDateString()  (see line 43)?

Reply:
Well as far as I understood last time I tried that it printed out the date in the wrong format (the US format), so I'm manually doing it to make sure it's in the correct format.

Reply:
Ah so this is the Irish format. Sorry I was getting confused as the list date format and the item view date format are different. See attached images:

![screenshot_20180618-204229](https://user-images.githubusercontent.com/27377996/41561645-60890990-7342-11e8-809f-1b0257ad1320.png)
![screenshot_20180618-204242](https://user-images.githubusercontent.com/27377996/41561658-675ac3da-7342-11e8-883f-ad28379047df.png)

Reply:
Got it, I'll merge this pull and update the times for both to be identical in the next pull request.